content/page/instructors.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Our instructors"
+date: 2020-01-08T07:39:57+01:00
+draft: true
+---
+
+This page lists all the instructors who have consented to be listed publicly on
+our web site so far. If you are a badged Instructor whose details do not appear
+here, but you would like to be listed, please propose a
+[merge request](https://github.com/paris-swc/paris-swc.github.io).
```

Comment:
I guess this is a Pull Request in github jargon.

Reply:
Good to me. I fixed it on a6e2ef7